language/proto/known_go_imports.go

```diff
@@ -6,6 +6,7 @@ package proto
 import "github.com/bazelbuild/bazel-gazelle/label"
 
 var knownGoProtoImports = map[string]label.Label{
+
```

Comment:
extra empty line. Is it generated?

Reply:
I believe this came from the "Update hardcoded proto import resolution tables" step from the https://github.com/bazelbuild/bazel-gazelle/wiki/Release-process. I did not edit it manually.